app/electron/main.ts

```diff
@@ -383,6 +402,63 @@ function startElecron() {
     });
 
     if (!useExternalServer) {
+      const runningHeadlamp = await getRunningHeadlampPIDs();
+      if (!!runningHeadlamp) {
+        const resp = dialog.showMessageBoxSync(mainWindow, {
+          // Avoiding mentioning Headlamp here because it may run under a different name depending on branding (plugins).
+          title: i18n.t('Other process running'),
+          message: i18n.t(
+            'Looks like another process is already running. Continue by stopping that process automatically, or quit?'
```

Comment:
To someone who doesn't know how Headlamp works, probably they need to know Headlamp runs a background process.

Reply:
I think that the info that "another process is running" in the context of Headlamp (or whatever branding) is less confusing than talking about background processes.

Reply:
"Continue by stopping" confused me at first.

I'm not sure if "Continue by terminating..." would be better.

["Terminate and Continue"] button?

hrmm. I'm not sure.

Reply:
I have replaced stopping by terminating. I kept the buttons because it's confusing and long to have "Terminate and continue"